docs/admin/authentication.md

```diff
@@ -33,7 +33,9 @@ plane, must authenticate when making requests to the the API server.
 ## Authentication strategies
 
 Kubernetes uses client certificates, bearer tokens, or HTTP basic auth to
-authenticate API requests through authentication plugins. As HTTP request are
+authenticate API requests through authentication plugins. A bearer token
+must be something that can be included in an HTTP header value without any
```

Comment:
The pattern for passing a bearer token appears in the doc for some of the four cases, and it is the same every time.  As a reader I felt it was pretty obvious that the same pattern would be followed in the cases where it was not exhibited.  My question was whether extra encoding, such as base64 would be needed.  I think the words I added answer that question.

I would be willing to submit a revised PR that shows the pattern for passing a bearer token once, up front so it applies in all four cases.  That would be a little difficult to fit into the existing outline: to be this expansive deserves a section of its own, and flow-wise that should come after the start of `## Authentication strategies` and before any of the `### <particular strategy>`, I suppose as another `###` in there.  Do you think this would be the right thing to do?